app/test_main.py

```diff
@@ -1,3 +1,24 @@
 from app.main import get_human_age
+import pytest
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age, result",
+    [
+        (0, 0, [0, 0]),
+        (14, 14, [0, 0]),
+        (15, 15, [1, 1]),
+        (23, 23, [1, 1]),
+        (24, 24, [2, 2]),
+        (27, 27, [2, 2]),
+        (28, 28, [3, 2]),
+        (100, 100, [21, 17]),
+    ]
+)
+def test_should_convert_animal_years_to_human(
+    cat_age: int, dog_age: int, result: list
+) -> None:
+    assert (get_human_age(cat_age, dog_age) == result), (
+        "'get_human_age' should return right "
+        "list with converted age"
+    )
```

Comment:
Please read [checklist.md](https://github.com/mate-academy/py-cat-and-dog-years/blob/master/checklist.md)